### DISCUSSION TITLE
Audio

### DISCUSSION THREAD

**Discussion by Mr-DaveDev** [Motion-Project/motionplus#64] [Enhancements]
The following sample code is desired from the community:
-  Sample code demonstrating the capture of audio via the web control and sending back to MotionPlus.  (Assume that browser is opened on a non local device)
- Sample code sending the audio stream from the netcam module to the web control.  (Processing a movie file with sound through the netcam_url via file option would demonstrate the same.)

**Comment by hyc** [comment]
Are there any more detailed documents anywhere on configuring for a netcam with audio? It seems the main docs for sound parameters all assume on-board audio hardware, and don't really apply to audio from a netcam.

What's the most common transport format for a net camera with audio+video? I'm working with an A9 mini wifi camera which returns data over UDP. I wrote https://github.com/hyc/a9serv to repackage the mjpg stream over an HTTP connection, and it works well with motion but I'm curious about how to add support for its audio channel.
